Vite test failure: assets.spec.ts build watch raw query import

## CI Failure

https://github.com/rolldown/rolldown/actions/runs/23382731682/job/68024743840

## Failing Test

`playground/assets/__tests__/assets.spec.ts > css and assets in css in build watch > import with raw query`

**Suite**: `test-build`

## Error

```
AssertionError: expected 'foo' to be 'zoo' // Object.is equality

Expected: "zoo"
Received: "foo"

 ❯ playground/assets/__tests__/assets.spec.ts:721:50
    719|     await notifyRebuildComplete(watcher)
    720|     await page.reload()
    721|     expect(await page.textContent('.raw-query')).toBe('zoo')
       |                                                  ^
    722|   })
    723| })
```

Build watch does not pick up file changes for raw query imports — the content remains `'foo'` instead of updating to `'zoo'`.

## Workaround

Skipped in #8840.

--- a/packages/vite-tests/run.ts
+++ b/packages/vite-tests/run.ts
@@ -76,6 +76,15 @@ await runCmdAndPipeOrExit(
   ['pnpm', ['run', 'build'], { nodeOptions: { cwd: REPO_PATH } }],
 );
 
+// Skip known failing tests
+// https://github.com/rolldown/rolldown/issues/8839
+const assetsSpecPath = path.resolve(REPO_PATH, 'playground/assets/__tests__/assets.spec.ts');
+const assetsSpec = fs.readFileSync(assetsSpecPath, 'utf-8');
+fs.writeFileSync(assetsSpecPath, assetsSpec.replace(
+  "test('import with raw query'",
+  "test.skip('import with raw query'"
+), 'utf-8');
+
 // Remove VITE_PLUS_* env vars to prevent leaking into loadEnv() test snapshots
 for (const key of Object.keys(process.env)) {
   if (key.startsWith('VITE_PLUS_')) {

